src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1512,3 +1512,42 @@ def list_consumption_locations():
     client = web_client_factory()
     regions = client.list_geo_regions(sku='Dynamic')
     return [{'name': x.name.lower().replace(" ", "")} for x in regions]
+
+
+def enable_zip_deploy(resource_group_name, name, src, slot=None):
+    # if src is not of .zip extension throw exception
+    if not src.endswith('.zip'):
```

Comment:
Case insensitive.

Reply:
Will be in v3

---
In reply to: [145752179](https://github.com/Azure/azure-cli/pull/4700#discussion_r145752179) [](ancestors = 145752179)

Reply:
My 2 cents, since we are uploading the file content, I don't think it is very necessary to check the zip file extension unless kudu only accepts one single compression mode.